BUILD.md

```diff
@@ -1,12 +1,24 @@
+#### Normalista uses [Github Pages](https://help.github.com/articles/using-jekyll-as-a-static-site-generator-with-github-pages/) to make easy to mantain and publish academic content.
+
+The stack is based on gulp and jekyll.
+
 ## Installation
 
-Install jekyll following http://jekyllrb.com/docs/installation/
+Install [jekyll](http://jekyllrb.com/docs/installation/).
+
+Install [npm](https://docs.npmjs.com/getting-started/installing-node) via node.
+
+Install [gulp](https://github.com/gulpjs/gulp/blob/master/docs/getting-started.md).
+
+Run `npm install`.
+
+Run `gulp`.
```

Comment:
If you add gulp in your devDependencies you can have a npm script, say `run` as :`"run": "gulp"`